cmd/clusterctl/examples/openstack/provider-component/user-data/centos/templates/worker-user-data.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Why updating here?

Reply:
because for different OS or distruibution, the bash may under different path